lib/stages/BadInputFlag.hpp

```diff
@@ -0,0 +1,72 @@
+/*****************************************
+@file
+@brief Extract a subset of products from a VisBuffer.
```

Comment:
copypasta

Reply:
I think I've already pushed a fix for this one.

Reply:
Sorry. Misread. No I haven't. Fixing now.